README.md

```diff
@@ -1,2 +1,15 @@
 # flooddrake
 Firedrake DG shallow water model with wetting and drying for flood simulation
+
+## Linux terminal installation
+As a prerequisite, it is required to have Firedrake installed alongside it's dependencies. Instructions on how to this can be found at: http://firedrakeproject.org/download.html.
+
+To install, type the following commands into the terminal whilst in an empty repository:
+
+1. `git init`
+2. `git clone https://github.com/firedrakeproject/flooddrake flooddrake`
```

Comment:
You don't necessarily need the last argument (flooddrake) as that will be the default directory name anyway, as it matches the last part of the URL.